### PR TITLE
[pvc_exporter] add alerts

### PR DESCRIPTION
**Pull request by rajivmucheli**
Hi @galkindmitrii @defo89 

If this PR is acceptable, we need to create a playbook.

Regards,
Rajiv

### CODE REVIEW

**Comment by galkindmitrii** [comment]
Looking good!
I think there are no playbooks for Info level alerts, or, are you planning to promote it to Warning later on?

**Comment by rajivmucheli** [comment]
if alerting around 70% is acceptable then severity INFO is good, if its 90% then severity WARNING is preferable
.

**Comment by defo89** [comment]
Not sure if I am missing something, but I think current template will not deploy alerts. There should be a `PrometheusRule` that will pick up the the alerts defined in `alerts/pvc_usage.alerts` folder ([example](https://github.com/sapcc/helm-charts/blob/master/system/kube-proxy/templates/alerts.yaml)).

I think `info` severity is a good start. For other severities we need to think how and by whom the alerts should be actioned (playbook).

**Comment by auhlig** [comment]
What @defo89 said^
Alerts are deployed via `PrometheusRule` CRD with label `prometheus: "kubernetes"` set in this case

**Review by Carthaca — CHANGES_REQUESTED** [review]

File: prometheus-exporters/pvc-exporter/alerts/pvc_usage.alerts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- name: pvc_usage.alerts
+  rules:
+  - alert: PVCUsagePercentageIsAbove70
+    expr: sum((pvc_usage* 100 > 70) and (predict_linear(pvc_usage[1d], 7 * 24 * 3600) * 100 > 90 )) by (persistentvolumeclaim, mountedby, volumename)
+    for: 1h
+    labels:
+      tier: k8s
+      service: resources
+      severity: info
+      context: memory
```
Comment (Carthaca) [inline prometheus-exporters/pvc-exporter/alerts/pvc_usage.alerts:11]: maybe `space` or `storage`

**Review by Carthaca — COMMENTED** [review]
Thanks, I just have some small description suggestions

**Review by defo89 — APPROVED** [review]
lgtm

**Review by Carthaca — APPROVED** [review]
👍

**Pull request merged**